code/modules/admin/verbs/randomverbs.dm

```diff
@@ -229,14 +229,14 @@ Allow admins to set players to be able to respawn/bypass 30 min wait, without th
 Ccomp's first proc.
 */
 
-/client/proc/get_ghosts(var/notify = 0,var/what = 2)
+proc/get_ghosts(var/notify = 0,var/what = 2)

```

Comment:
Refractoring this into a global proc will create an issue at line 245 of this file. It'll attempt to access `src` (the client) there, but for a global proc, there is no natural `src` variable.

You could change this to accept an optional parameter of `var/client/C = null`, and go from there:
`/proc/get_ghosts(var/what = 2, var/notify = 0, var/client/C = null)`

The `if(notify)` check would then be like this, instead:
```
if (notify && C)
    C << "There doesn't appear to be any ghosts for you to select."
```